### ISSUE TITLE
No print to Output panel in infinite loop

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#22184]
Rephrasing issue #1287 which has become hard to read given the various other debugging issues which have been raised (and fixed) there.

**Godot version:**
All up to current master branch (ebd14a77).

**OS/device including version:**
Linux, likely other OSes too.

**Issue description:**
When using a `print` statement in an infinite loop, nothing will be printed to the Output panel. There are however prints to the system console/stdout.

**Steps to reproduce:**
Run a script with:
```
func _ready():
    var i = 0
    while true:
        i += 1
        print(i)
```
No text in Output panel. Terminal shows:
```
1
2
3
...
281694
281695
281696
281697
...
```

**Comment by DeuxAlpha** [comment]
Reproducible on Windows 10. Also, the type of loop does not matter, for and while-loop provide the same behavior.

**Comment by rogeriodec** [comment]
Same here...
Godot 3.1 official
Windows 10 x64

```
.
.
.
********pos_atu_eixo: -0.961239, meio eixo:1.698761
********pos_atu_eixo: -1.241239, meio eixo:1.698761
********pos_atu_eixo: -1.521239, meio eixo:1.698761
********pos_atu_eixo: -1.801239, meio eixo:1.698761
-----angulo_atu: -109.571818
********pos_atu_eixo: -1.801239, meio eixo:1.698761
-----angulo_atu: -163.047879
********pos_atu_eixo: -1.801239[...]
[output overflow, print less text!```

How to solve this?

**Comment by KoBeWi** [comment]
Seems to be fixed since at least 3.1.2.

**Comment by Gameoholic** [comment]
Same issue is present in version 3.4.2
